datasets/rdw/rdw.json

```diff
@@ -0,0 +1,90 @@
+{
+  "type": "dataset",
+  "id": "rdw",
+  "title": "RDW",
+  "status": "beschikbaar",
+  "description": "De RDW is de organisatie die de registratie van gemotoriseerde voertuigen en rijbewijzen in Nederland verzorgt. De RDW is een zelfstandig bestuursorgaan van de Nederlandse overheid.",
+  "version": "0.0.1",
```

Comment:
Could you add a `contactPoint` for this dataset on this level (toplevel)? It will (eventually) be used to email the `contactPoint` should the import fail.

```json
    "contactPoint": {
      "type": "object",
      "properties": {
        "name": {"type": "string"},
        "email": {"type": "string"}
      }
```

Reply:
Check!